content/docs/dvclive/user-guide/dvclive-with-dvc.md

```diff
@@ -0,0 +1,104 @@
+# DVCLive with DVC
+
+Even though DVCLive does not require DVC, they can integrate in several useful
+ways.
+
+The logs/summaries DVCLive produces can be fed as `dvc plots`/`dvc metrics`.
+
+## Setup
+
+We will refer to a training script (`train.py`) already using `dvclive`:
+
+```python
+# train.py
+
+import dvclive
+
+for epoch in range(NUM_EPOCHS):
+    train_model(...)
+    metrics = evaluate_model(...)
+
+    for metric_name, value in metrics.items():
+        dvclive.log(metric_name, value)
+
+    dvclive.next_step()
+```
+
+> 💡 When using DVCLive in a DVC project, there is no need for manual
+> initialization of DVCLive (no [`dvclive.init()`] call). DVCLive will use the
+> settings provided by DVC.
+
+Let's use `dvc stage add` to create a stage to wrap this code (don't forget to
+`dvc init` first):
+
+```dvc
+$ dvc stage add -n train --live training_metrics
+                -d train.py python train.py
+```
+
+`dvc.yaml` will contain a new `train` stage with the DVCLive
+[configuration](/doc/dvclive/api-reference/init#parameters) (in the `live`
+field):
+
+```yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+      - train.py
+    live:
+      training_metrics:
+        summary: true
+        html: true
+```
+
+The value passed to `--live` (`training_metrics`) became the directory `path`
+for DVCLive to write logs in, and DVC will now
+[track](/doc/use-cases/versioning-data-and-model-files) it. Other supported
+command options for the DVC integration:
+
+- `--live-no-cache <path>` - specify a DVCLive log directory `path` but don't
+  tracked it with DVC. Useful if you prefer to track it with Git.
+- `--live-no-summary` - passes `summary=False` to DVCLive.
+- `--live-no-html` - passes `html=False` to DVCLive.
+
+> Note that these are convenience CLI options. You can still use
+> [`dvclive.init()`] manually, which will override any options sent to
+> `dvc stage add`. Just be careful to match the `--live` value (CLI) and `path`
+> argument (code). Also, note that summary files are never tracked by DVC
+> automatically.
+
+Run the training with `dvc repro`:
+
+```bash
+$ dvc repro train
+```
+
+## Outputs
+
+After that's finished, you should see the following content in the project:
+
+```bash
+$ ls
+dvc.lock  training_metrics       training_metrics.json
+dvc.yaml  training_metrics.html  train.py
+```
+
+In addition to the
+[outputs described in the Quickstart](/docs/dvclive/user-guide/quickstart#outputs),
+DVC generates an _HTML report_. If you open `training_metrics.html` in a
+browser, you'll see a plot for metrics logged during the model training!
+
+![](/img/dvclive_report.png)
+
+## Checkpoints
```

Comment:
I think checkpoints are the primary usage for integration and should be introduced sooner.

Now that we have some support for saving models in dvclive, maybe we need deeper integration so that `dvc stage add --live` could manage the model file path and add checkpoints automatically?

Reply:
I think it's a good idea but it would require to rethink how we handle models on the `dvclive` side. 

Currently, setting `path` on the `dvc` side (`dvc stage add --live {path}`) makes easier to assume things like `{path}/*.tsv` are plots and `{path}.json` are metrics. 

However, the `model_file` that indicates where the model will be saved is currently being handled on the "integration" side (i.e. as an argument to `keras` Callback). 

If we want to start managing the model file from the `dvc` side we would probably need to make `model_file` a `dvclive.init` argument  & `DVCLIVE_` env variable, right?

Reply:
This seems to be getting back to the discussion in https://github.com/iterative/dvclive/issues/69 and https://github.com/iterative/dvclive/issues/105, so let's keep discussing there. I wanted to raise it before I forgot, but it's out of scope for this PR, so no need to block it.